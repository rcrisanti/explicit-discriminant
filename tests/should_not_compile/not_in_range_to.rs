use explicit_discriminant::ExplicitDiscriminant;

#[derive(ExplicitDiscriminant)]
#[pattern(..3)]
enum Test {
    Zero = 0,
    One = 1,
    Two = 2,
    Three = 3,
}

fn main() {
    let t = Test::One;
}
