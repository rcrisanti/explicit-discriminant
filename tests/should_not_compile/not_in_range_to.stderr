error: discriminant does not match any pattern
 --> tests/should_not_compile/not_in_range_to.rs:9:13
  |
9 |     Three = 3,
  |             ^
